### ISSUE TITLE
Player Experience Road

### ISSUE THREAD

**Issue by QuinnBast** [Subterfuge-Revived/Remake-Frontend#7]
As players play games, they should gain 'experience' based on their total Np mined. As players level up various unlocks should be able to be recieved. These unlocks should be designed to be locked until obtained and once obtained they may alter game effects such as the number of scheduled events, number of games the player can play at once, or unlock specialists. Some may be available to the player for customization before they enter a game (?).

**Comment by Eliclax** [comment]
I don't think we should limit scheduled events, and if there is a rank-up system, I think the full standard game should be available to them at the start of the game. I do like the idea of Total mined Np being the experience though — that might fit in well with the lore too.
